custom_hooks/safety_check.sh

```diff
@@ -13,4 +13,4 @@ if ! command -v safety &> /dev/null; then
     pip install safety
 fi
 
-poetry export --without-hashes -f requirements.txt | safety check --full-report --stdin
+poetry export --without-hashes -f requirements.txt | safety check --full-report --stdin --ignore=70612
```

Comment:
could we add a comment just explaining why we are ignoring 70612 just so we arent confused down the road

Reply:
cheers